tests/menus/menu_items_registry.test.ts

```diff
@@ -929,6 +929,59 @@ describe("Menu Item actions", () => {
     restoreDefaultFunctions();
   });
 
+  describe("Insert -> Checkbox", () => {
+    test("In an empty cell", () => {
```

Comment:
I'd move all those tests with the DV plugins tests (either core or evaluation, there's no obvious place since it involves both). Most of them are testing edge cases which are handled inside the plugins, and they are not specific to the menu item in the registry.
So instead of `doAction` => use `addDataValidation` like other tests in those files.

You can leave only one simple test case here to test the menu item itself.

EDIT: actually, the file `data_validation_checkbox_plugin.test.ts` is the perfect file ;)

Also, the file `data_validation_checkbox_component.test.ts` does not test the component and seem a vague copy-pasted version of the other file.